erpnext/stock/doctype/stock_entry/stock_entry.js

```diff
@@ -494,6 +506,7 @@ frappe.ui.form.on('Stock Entry Detail', {
 	qty: function(frm, cdt, cdn) {
 		frm.events.set_serial_no(frm, cdt, cdn, () => {
 			frm.events.set_basic_rate(frm, cdt, cdn);
+			frm.events.calculate_total_fields(frm, cdt, cdn);
```

Comment:
Do we need to call this for item_code and items_remove events as well? Did you see Sundine's total qty implementation?